presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -775,6 +775,54 @@ Now, Presto queries can take advantage of the Alluxio catalog service, such as
 transparent caching and transparent transformations, without any modifications
 to existing Hive metastore deployments.
 
+Source Data Caching
+-------------------
+
+Presto can cache data read from remote sources, e.g. cloud stores, on the cluster to speed up the subsequent reads.
+Data caching can be enabled by setting the following properties:
+
+============================================ ======================================================================
+Property Name                                Description
+============================================ ======================================================================
+``hive.cache.enabled``                       Enables data caching when set to ``true``
+
+``hive.cache.location``                      The location on nodes to use to store the cached data.
+                                             In-memory caching can be achieved by providing RAM disk location here.
+                                             For on-disk caching, it is recommended to use SSD disks
+============================================ ======================================================================
+
+Setting these configs for a hive catalog enables data caching on all tables in that catalog.
+Following optional configurations can be used to control the caching behavior:
+
+============================================ ================================================================== ===========
+Property Name                                Description                                                          Default
+============================================ ================================================================== ===========
+``hive.cache.parallel-warmup-enabled``       Enables asynchronous cache warmup to minimize the impact of        ``true``
+                                             cold-cache.
+
+``hive.cache.bookkeeper-port``               Defines the port to be used by the BookKeeper server.              8899
```

Comment:
A comment that if caching is enabled for multiple catalogs, each catalog should provide a different value for the cache location and the bookkeeper and data-transfer ports will also resolve https://github.com/prestosql/presto/issues/3723.